netlify/edge-functions/og-image.tsx

```diff
@@ -0,0 +1,245 @@
+import React from 'https://esm.sh/react@18.2.0';
+// eslint-disable-next-line import/extensions
+import { ImageResponse } from 'https://deno.land/x/og_edge/mod.ts';
+
+const MAX_AUTHORS = 5;
+export default async function handler(req: Request) {
+  const params = new URL(req.url).searchParams;
+  const title = params.get('title');
+  const authors = params.get('authors');
+  const product = params.get('product');
+
+  const parsedAuthors =
+    authors &&
+    authors
+      .split(',')
+      .map((author) => {
+        const [name, github] = author.split('@');
+        return { name: name.trim(), github: github.trim() };
+      })
+      .filter(({ name, github }) => name && github);
+
+  const withAuthors = parsedAuthors && parsedAuthors.length > 0;
+  let starCount = 0;
+
+  return new ImageResponse(
+    (
+      <div
+        style={{
+          width: '100%',
+          height: '100%',
+          background: 'linear-gradient(180deg, #FFFFFF 0%, #F0F7FF 75.52%)',
+          overflow: 'hidden',
+          position: 'relative',
+          display: 'flex',
+        }}
+      >
+        <div
+          style={{
+            marginTop: 100,
+            marginLeft: 100,
+            marginRight: 100,
+            marginBottom: 100,
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'space-between',
+          }}
+        >
+          <div
+            style={{
+              height: 40,
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+            }}
+          >
+            <svg
```

Comment:
Another quick design nudge here 😅 Maybe we want to make this logo rendering conditional, too? At the moment, we only have the MUI/Material UI (company/project) logo. But we'll have product-specific logos soon (like, one for Base UI, Pigment CSS, etc.), and they should probably be rendered here if it's their documentation space. Is it possible?

Reply:
Yes, it's feasible, but it looks easier to first have the other logos, and then update the code

Reply:
Makes sense—at least you have this in mind already now 😃